<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::Internal::CompletionSettingsPage</class>
 <widget class="QWidget" name="CppTools::Internal::CompletionSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>376</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="spaceAfterFunctionName">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Insert &amp;space after function name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="caseSensitivityLabel">
        <property name="text">
         <string>&amp;Case-sensitivity:</string>
        </property>
        <property name="buddy">
         <cstring>caseSensitivity</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="caseSensitivity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Full</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First Letter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="surroundSelectedText">
          <property name="toolTip">
           <string>When typing a matching character and there is a text selection, instead of removing the selection, surrounds it with the corresponding characters.</string>
          </property>
          <property name="text">
           <string>Surround &amp;text selections</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="autoSplitStrings">
        <property name="toolTip">
         <string>Splits a string into two lines by adding an end quote at the cursor position when you press Enter and a start quote to the next line, before the rest of the string.

In addition, Shift+Enter inserts an escape character at the cursor position and moves the rest of the string to the next line.</string>
        </property>
        <property name="text">
         <string>Automatically split strings</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" colspan="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="completionTrigger">
        <item>
         <property name="text">
          <string>Manually</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>When Triggered</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="automaticProposalTimeoutLabel">
        <property name="text">
         <string>Timeout in ms:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="partiallyComplete">
        <property name="toolTip">
         <string>Inserts the common prefix of available completion items.</string>
        </property>
        <property name="text">
         <string>Autocomplete common &amp;prefix</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="completionTriggerLabel">
        <property name="text">
         <string>Activate completion:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="autoInsertBrackets">
        <property name="toolTip">
         <string>Automatically inserts semicolons and closing brackets, parentheses, curly braces, and quotes when appropriate.</string>
        </property>
        <property name="text">
         <string>&amp;Automatically insert matching characters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="automaticProposalTimeoutSpinBox">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="docCommentsGroup">
     <property name="title">
      <string>Documentation Comments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="enableDoxygenCheckBox">
        <property name="toolTip">
         <string>Automatically creates a Doxygen comment upon pressing enter after a '/**', '/*!', '//!' or '///'.</string>
        </property>
        <property name="text">
         <string>Enable Doxygen blocks</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="generateBriefCheckBox">
          <property name="toolTip">
           <string>Generates a &lt;i&gt;brief&lt;/i&gt; command with an initial description for the corresponding declaration.</string>
          </property>
          <property name="text">
           <string>Generate brief description</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="leadingAsterisksCheckBox">
        <property name="toolTip">
         <string>Adds leading asterisks when continuing C/C++ &quot;/*&quot;, Qt &quot;/*!&quot; and Java &quot;/**&quot; style comments on new lines.</string>
        </property>
        <property name="text">
         <string>Add leading asterisks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autoInsertBrackets</sender>
   <signal>toggled(bool)</signal>
   <receiver>spaceAfterFunctionName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoInsertBrackets</sender>
   <signal>toggled(bool)</signal>
   <receiver>surroundSelectedText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableDoxygenCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>generateBriefCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
